testing/build.gradle

```diff
@@ -102,6 +102,7 @@ dependencies {
       'com.google.truth:truth:1.1.3',
       'junit:junit:4.12',
       "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version",
+      "org.jetbrains.kotlin:kotlin-test",
```

Comment:
Why was this added? It is my understanding that nothing has actually changed in the test implementation.